### ISSUE TITLE
Limited reach of OUTCAR parameters

### ISSUE THREAD

**Issue by ndaelman-hu** [nomad-coe/electronic-parsers#4]
In `OutcarTextParser` body, the `parameter` extracts keywords up to 100 lines after the starting keyword.
A DFT+U test scheduled to be added, supersedes this limit, however.
Rather than just increasing the number of lines, could we instead add a _stop flag_ to denote the end of the section?

**Comment by ladinesa** [comment]
I do not quite understand this parameter. Are you referrring to this 

```
            Quantity(
                'parameters', r'Startparameter for this run:([\s\S]+?)\-{100}',
                str_operation=get_key_values, repeats=False, convert=False),
```

**Comment by ndaelman-hu** [comment]
Yes: more specifically, I'm encountering (test) cases where the 100 limit isn't sufficient.

**Comment by ladinesa** [comment]
This is the end of section regex that I see from the files that I encountered. If you have another you can also add it like this:

```
            Quantity(
                'parameters', r'Startparameter for this run:([\s\S]+?)(?:\-{100}:__your_regex)',
                str_operation=get_key_values, repeats=False, convert=False),
```
Just be sure that this regex is not found before the 100 hyphens. You can also maybe perhaps reduce the number of hyphens to match your case?
